ovirt-ansible-hosted-engine-setup.spec.in

```diff
@@ -15,7 +15,11 @@ BuildArch:      noarch
 Url:            http://www.ovirt.org
 
 Requires: ansible >= 2.7
+%if 0%{?rhel} >= 8
```

Comment:
you should also target fedora >=30 to python3-netaddr

Reply:
thanks, done